src/components/FilterPanel/FilterPanel.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { describe, it, expect, vi } from 'vitest';
+
+import { FilterPanel } from './FilterPanel';
+
+describe('FilterPanel', () => {
+  it('resalta el filtro seleccionado', () => {
+    render(
+      <FilterPanel
+        filters={[
+          { label: 'All Tests', value: 'all' },
+          { label: 'Upcoming', value: 'upcoming' },
+        ]}
+        selectedValue="all"
+        onSelect={() => {}}
+      />
+    );
+
+    const selectedButton = screen.getByTestId('filter-all');
+    expect(selectedButton).toHaveAttribute('aria-checked', 'true');
+  });
+
+  it('dispara onSelect al hacer clic en un filtro', () => {
```

Comment:
Second test title is in Spanish. Also tests should assert behavior using accessible selectors.

**Sugerencia:**
```typescript
Translate and use semantic selectors. Example:

it('calls onSelect when a filter is clicked', async () => {
  const mockSelect = vi.fn();
  render(...)
  const upcoming = screen.getByRole('radio', { name: /Upcoming/i });
  await userEvent.click(upcoming);
  expect(mockSelect).toHaveBeenCalledWith('upcoming');
});

(Prefer user-event over fireEvent for user interactions.)
```